style.css

```diff
@@ -0,0 +1,619 @@
+
+html {
+  overflow-x: hidden;
+}
+
+/* Top Navbar */
+
+.navbar-container {
+  display: flex;
+  width: 100%;
+}
+
+.menu-container {
+  width: 100%;
+}
+
+.logo {
+  width: 66px;
+  height: 16px;
+}
+
+.logo-container {
+  padding: 14px 20px;
+  background-color: black;
+  box-sizing: border-box;
+}
+
+/* Black Navbar */
+
+.black-menu-container {
+  display: none;
+  border: 1px solid black;
+  background-color: black;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.right-black-menu,
+.left-black-menu {
+  display: flex;
```

Comment:
**[OPTIONAL]**
* [ ] Avoid redefining a same attribute over and over, you define it, in isolation and used wherever you want to. We could declare this as a class on it own. 
`.flex{
display: flex;
}
`